src/services/inlayHints.ts

```diff
@@ -283,6 +291,15 @@ namespace ts.InlayHints {
 
                 addTypeHints(typeDisplayString, param.end);
             }
+
+            if (wrapParen) {
+                // ensure ")" is added after paremeter hints.
+                result.push({
+                    text: ")",
+                    position: node.parameters[node.parameters.length - 1].end,

```

Comment:
We just want add paren hints when only 1 argument and without parents

Reply:
updated

Reply:
There's a `last` helper function you can use.